binary_transparency/firmware/cmd/ft_witness/internal/ws/store.go

```diff
@@ -64,12 +63,12 @@ func (ws *Storage) StoreCP(wcp []byte) error {
 	// Check if file exists, open for write and store the checkpoint
 	f, err := os.OpenFile(ws.fp, os.O_RDWR, fileMask)
 	if err != nil {
-		f.Close()
+		_ = f.Close()
```

Comment:
Hmm, not your change, but this file-handling code seems a bit unusual in here, e.g.:
 - is `f` guaranteed to be non-nil if `err` is non-nil?
 - I wonder why we're doing `os.OpenFile`/`f.Write`/`f.Close` rather than `os.WriteFile`?
 
No need to fix in your PR, but we should probably make a note and look at this...

Reply:
Addressed in #799